testapi.pm

```diff
@@ -1539,7 +1539,7 @@ sub eject_cd {
 
 =head2 save_memory_dump
 
-  save_memory_dump(filename => undef, migration_speed => "4096m");
+  save_memory_dump([$filename, $max_downtime]);
 
 Saves the SUT memory state using C<$filename> as base for the memory dump
 filename,  the default will be the current test's name.
```

Comment:
I recommend to mention the default `max_downtime` in the description. Yes, it's annoying that we have to repeat that but if we insist on using perl we are stuck with POD then IMHO we have to do that